README.md

```diff
@@ -1 +1,63 @@
-# ironfoudation-team-project
+# WOOF application 
+
+## Table of contents
+* [General info](#general-info)
+* [Technologies](#technologies)
+* [Installation](#installation)
+* [Authors](#authors)
+* [Notes](#notes)
+
+## General info
+
+This is the application for iOS.
+Woof application connects dog owners with pet sitters which contributes to build trusted relationships between them. The app makes it easy for nearby pet owners to find pet sitting and makes convienience of responding to jobs for pet sitters from phone.
+App also makes managing sitting bussines and pet owner's schedule. They can choose their own schedule, prices, and requirements for servise. 
+
+**Features include**
+- Secure store all of pet owners and pet sitters databases.
+- Regularly database update with personal information.
+
+**It also important to know:**
+To process the application some personal information about customers is required, for example, home address, mobile phone.
+
+## Technologies
+
+To install the application, please, see the list of requirements and restrictions settings below:
+ * Deployment target for iOS SDKs - 15.0
```

Comment:
Do I need Xcode to install app? Is that a requirements to install app?

Reply:
What version of macOS should I have?

Reply:
Yes, Xcode is needed. 
macOS Ventura 13.3+